hokusai/development.yml

```diff
@@ -2,16 +2,18 @@ version: "2"
 services:
   metaphysics:
 {% include 'templates/docker-compose-service.yml.j2' %}
-    environment:
-      - MEMCACHED_URL=metaphysics-memcached:11211
+    command: ["yarn", "start"]
     env_file:
       - ../.env.shared
+      - ../.env.development
       - ../.env
-    ports:
-      - 3000:3000
     depends_on:
       - metaphysics-memcached
+    ports:
+      - 5001:5001
+    volumes:
+      - ../:/app
```

Comment:
I decided to mount the volume here. Will need to see if this has any performance implications.